slurm/evaluate.slurm

```diff
@@ -8,6 +8,9 @@
 #SBATCH --time=01:59:00
 #SBATCH --output=./logs/evaluate/%x-%j.out
 #SBATCH --err=./logs/evaluate/%x-%j.err
+#SBATCH --mem=128G
```

Comment:
These are specific to your cluster and do not match the specs of ours so I would not include them.